camel-example-spring-boot-actuator-http-metrics/src/main/java/sample/camel/MyRouteBuilder.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sample.camel;
+
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.model.RouteDefinition;
+import org.apache.camel.model.rest.RestBindingMode;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MyRouteBuilder extends RouteBuilder {
+
+    @Override
+    public void configure() throws Exception {
+
+        // First, we have to configure our jetty component, which will be the rest
+        // in charge of querying the REST endpoints from actuator
+        restConfiguration().component("jetty")
+                .host("0.0.0.0")
+                .port(8080)
+                .bindingMode(RestBindingMode.json);
+
+        // First, let's show the routes we have exposed.
+        doOnce()
```

Comment:
Function `doOnce()` builds a `from("timer:queryTimer?repeatCount=1")` route definition,
but below there are 2 route definitions, that are defined explicitly:
1. `from("timer:metricsTimer?period={{metricsPeriod}}")`
2. `from("timer:shutdownTimer?delay={{shutdownTime}}&repeatCount=1")`

I think that that `doOnce` could be removed for consitency.

Reply:
Agreed. I used the doOnce function at first because I expected to use it as a fire for other routes, but it ended being used only in the mappings one. Although I like having well named functions, it really makes no sense when the others are built explicitly from properties Un_n.

Gonna remove it. Let me a moment

Reply:
Fixed on https://github.com/apache/camel-spring-boot-examples/pull/16/commits/3d62b7b657099614d9b8c01d0d5534d5b71f09ff